_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5524,12 +5525,13 @@ mfxStatus ConfigureExecuteParams(
                         if (videoParam.ExtParam[i]->BufferId == MFX_EXTBUFF_VPP_MIRRORING)
                         {
                             mfxExtVPPMirroring *extMirroring = (mfxExtVPPMirroring*) videoParam.ExtParam[i];
-                            if (extMirroring->Type != MFX_MIRRORING_DISABLED && extMirroring->Type != MFX_MIRRORING_HORIZONTAL)
+                            if (extMirroring->Type != MFX_MIRRORING_DISABLED && extMirroring->Type != MFX_MIRRORING_HORIZONTAL
+#if defined(MFX_VA_LINUX)
```

Comment:
I still would prefer to see no MFX_VA_LINUX macro in this code. Can we use runtime m_pCore->GetVAType() != MFX_HW_VAAPI here as we did above?

Reply:
As I explained before, m_pCore is invalid in ConfigureExecuteParams. How about filing another patch to pass m_pCore to ConfigureExecuteParams() and remove MFX_VA_LINUX macro?